py3status/modules/xrandr_rotate.py

```diff
@@ -34,7 +34,7 @@
     color_degraded: Screen is disconnected
     color_good: Displayed rotation is active
 
-@author Maxim Baz (https://github.com/maximbaz)
+@author Maxim Baz (https://github.com/maximbaz) @contributor lasers
```

Comment:
@lasers can you just change this
```
@author Maxim Baz (https://github.com/maximbaz), @lasers
```

Also can you add it __as a single extra commit__.   That way I'll merge right away.  If you add it in with the existing commit I'll need to review it all again which may not be for several days.